qb_upload_limit_per_day.py

```diff
@@ -8,17 +8,18 @@
 
 # Configuration
 UPLOAD_LIMIT = 50  # Upload limit in GB (per day)
-QB_URL = "http://localhost:8080"  # qBittorrent Web UI URL
-CHECK_INTERVAL = 10  # In seconds
+QB_URL = "http://localhost:8080" # qBittorrent Web UI URL
+CHECK_INTERVAL = 60  # In seconds
```

Comment:
```suggestion
QB_URL = "http://localhost:8080"  # qBittorrent Web UI URL
CHECK_INTERVAL = 10  # In seconds
```
</br>

> The default update time is increased to 60 seconds, as global statistics is updated less often than the session one.

Let the check interval be 10 seconds, it updates pretty often for me though. For people who has lot of files to seed and fast internet connection 60 seconds is too long as within a minute they can upload 1.5 GB of data.

![image](https://github.com/user-attachments/assets/e5f1d712-3adf-41fe-a1c4-7d9d3fbbf9f9)

Reply:
Sure, I think it depends on the qBittorrent version. On my side, it wasn't updating quicker than 1 every minute even with a heavy upload traffic.

Reply:
Ok fine if you are sure that it only updates once a minute then let that be `60`. As this script will currently work for v4.x.x. In future I will prob add support for v5.x.x. At that time I will change their check interval according to the version.

Reply:
Reverted back to 10 seconds, don't see any issues with that